src/components/pages/advanced.tsx

```diff
@@ -0,0 +1,81 @@
+import robot1 from '../../assets/robots/pg6/robot1pg6.svg';
+import robot2 from '../../assets/robots/pg6/robot2pg6.svg';
+import robot3 from '../../assets/robots/pg6/robot3pg6.svg';
+import robot4 from '../../assets/robots/pg6/robot4pg6.svg';
+import {lineOfCode, CodeFormat}  from  '../shared/codeFormat';
+import {Color} from '../shared/constants';
+
+function Advanced(): JSX.Element {
+
+  const code: lineOfCode[] = [
+    [
+      { ['print']: Color.Orange },
+      { ['(num_of_antennas']: Color.White },
+      { [' > ']: Color.Orange },
+      { ['1']: Color.Blue },
+      { [' or ']: Color.Orange },
+      { ['has_mouth)']: Color.White },
+    ],
+    [
+      { ['print']: Color.Orange },
+      { ['(color']: Color.White },
+      { [' == ']: Color.Orange },
+      { ['"red"']: Color.Green },
+      { [' and ']: Color.Orange },
+      { ['color']: Color.White },
+      { [' != ']: Color.Orange },
+      { ['"blue"']: Color.Green },
+      { [')']: Color.White },
+    ],
+    [
+      { ['print']: Color.Orange },
+      { ['(']: Color.White },
+      { ['not ']: Color.Orange },
+      { ['head_shape']: Color.White },
+      { [' == ']: Color.Orange },
+      { ['"circle"']: Color.Green },
+      { [')']: Color.White },
+
+    ],
+  ];
+
+  return (
+    <div className="frame">
+      <div id="advanced" className='robot-layout'>
+        <div className="code-and-output">
+          <div id='code'>
+            <CodeFormat code={code}/>
+          </div>
+          <div id="output">
+            <div className="output-items">
+              <div>True</div>
+              <div>False</div>
+              <div>True</div>
+            </div>
+
+          </div>
+        </div>
+
+        <h1>Select the matching robot:</h1>
+        <div className="robots">
+          <button className="robot-button">
+            <img src={robot1} alt="red robot, circle head, with mouth, 1 antenna, 2 arms, 1 button"></img>
+          </button>
```

Comment:
id recommend having a `Robot` component if you have time to separate it out! This robot component will deal with all the importing of images, and could be passed in something like
```
interface RobotProps {
   color: Color;
   headShape: Shape;
   hasMouth: boolean;
   numAntennas: number;
   numArms: number;
   numButtons: number;
}
```

This might take a bit of time since you would need to separate the robots by its attributes. I'd recommend having a `robots` folder where robots are named something like `robot-RCM121` and having a `README.md` that briefly mentions 
``` 
file names are based on the robot attibutes
robot-CS[M]###
C refers to robot color
S refers to robot shape
M refers to whether there is a mouth. no M means no mouth
#1 is # of antennas
#2 is # of arms
#3 is # of buttons
```

Reply:
The reason why this is better is because it makes it easier to change the exact details of the project in addition to making this more reusable in the future. In addition, it's more obviously visible what's going on in the code <which is very important since code is read much more than it is written>

Hard coding things works, but since we are doing react and not html/css it's good to leverage the react benefits!! I know you already did it this way, so I don't mind keeping it this way since it's not a small effort to change to this. Totes your choice

Reply:
@nendow02 What do you think would you like me to do this? It would take me a while probably, I'm not really familiar with interfaces and stuff in React. But it would be more reusable.

Reply:
I think we should save this for later. The other method would be more ideal but seeing that the way we have it now is perfectly functional I think its fine to leave it this way for now.